src/Transition/CreateNewIDFUsingRulesV22_1_0.f90

```diff
@@ -418,6 +418,13 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! If your original object starts with O, insert the rules here
 
               ! If your original object starts with P, insert the rules here
+              CASE('PYTHONPLUGIN:SEARCHPATHS')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 nodiff=.false.
+                 OutArgs(1:3)=InArgs(1:3)
+                 OutArgs(4) = 'Yes'
+                 OutArgs(5:CurArgs+1)=InArgs(4:CurArgs)
+                 CurArgs = CurArgs + 1
```

Comment:
Simple transition rule to increment the arg count by one and push everything past by one spot.